doobie-hikari-pureconfig/src/main/scala/com/avast/sst/doobie/pureconfig/ConfigReaders.scala

```diff
@@ -21,6 +21,15 @@ trait ConfigReaders {
     case unknown                        => Left(CannotConvert(unknown, "TransactionIsolation", "unknown value"))
   }
 
+  implicit val doobieTransactionIsolationWriter: ConfigWriter[TransactionIsolation] = ConfigWriter[String].contramap[TransactionIsolation] {
+    case TransactionIsolation.TransactionNone            => "TRANSACTION_NONE"
+    case TransactionIsolation.TransactionReadUncommitted => "TRANSACTION_READ_UNCOMMITTED"
+    case TransactionIsolation.TransactionReadCommitted   => "TRANSACTION_READ_COMMITTED"
+    case TransactionIsolation.TransactionRepeatableRead  => "TRANSACTION_REPEATABLE_READ"
+    case TransactionIsolation.TransactionSerializable    => "TRANSACTION_SERIALIZABLE"
+  }
+
   implicit val doobieDoobieHikariConfigReader: ConfigReader[DoobieHikariConfig] = deriveReader
+  implicit val doobieDoobieHikariConfigWriter: ConfigWriter[DoobieHikariConfig] = deriveWriter
```

Comment:
This can't be part of `ConfigReaders` trait, right? :)

Reply:
🤦‍♂️ of course!